Makefile

```diff
@@ -23,11 +22,10 @@ redeploy-container:
 deploy-container:
 	docker-compose run web sleep 5
 	docker-compose run web rake db:migrate
-	docker-compose run web bower install --allow-root
-	docker-compose run web bower update --allow-root
 	docker-compose run web rake assets:precompile
 	rm -f ./tmp/pids/server.pid
 	docker-compose up -d
+	docker-compose exec -T web yarn --ignore-engines --ignore-scripts --modules-folder public/lib 
```

Comment:
Again here, you wait till after deploying the container to install Javascript packages, I think the same applies here. Thanks!